ArticleTemplates/assets/js/modules/ads.js

```diff
@@ -177,9 +202,9 @@ define([
                     modules.insertAdPlaceholders(config);
                 }
                 window.getMpuPosJson = modules.getMpuPosJson;
-                window.getMpuPosCommaSeparated = modules.getMpuPosCommaSeparated;
-                window.getBannerPosCallback = modules.getBannerPosCallback;
-                modules.getAds();
+                window.getBannerPosCallback = modules.getBannerPosCallback; // Used by Android
+                modules.getAds(); // Used by Android
+                modules.updateAdsIos(); // Used by iOS
```

Comment:
Can we invoke this function only if we're sure the device is iOs ? Otherwise we'll end up having an extra polling routine running in our page